client/src/mock_api/handlers.ts

```diff
@@ -0,0 +1,123 @@
+import { delay, http, HttpResponse, passthrough } from "msw";
+
+const POKEMON_BASE_URL = "http://localhost:8080/api";
+
+const pokemonList = [
+  { id: 0, name: "andy" },
+  { id: 1, name: "bert" },
+  { id: 2, name: "charlie" },
+  { id: 3, name: "dave" },
+  { id: 4, name: "eric" },
+  { id: 5, name: "fred" },
+];
+
+const pokemonCards = [
+  {
+    id: 0,
+    name: "andy",
+    imageUrl:
+      "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/shiny/151.png",
+    description: "",
+    type: "psychic",
+    stats: [
+      { name: "attack", value: 100 },
+      { name: "defense", value: 100 },
+      { name: "speed", value: 100 },
+      { name: "weight", value: 40 },
+      { name: "hp", value: 100 },
```

Comment:
Are the value fields always whole numbers, there's non with decimals? I assume the code would handle fine as it's a `number`?

Reply:
Yeah the Pokemon numbers seemed to all be integer so that's what I stuck with - though I didn't do an exhaustive check.

And my head went into a spin because footie or ship or whatever values wouldn't necessarily be integer.

That was one of the reasons I ended up naming the hooks to usePokemonList/Card - each external API would have a different schema.